components/Layout/PageHeader.tsx

```diff
@@ -1,15 +1,56 @@
-import { Box, Heading } from '@chakra-ui/core'
 import * as React from 'react'
+import { Box, Link, Text, Heading, Icon, Image } from '@chakra-ui/core'
 
 interface PageHeaderProps {
   title: string
+  isHomePage?: boolean
+  welcomeMessage?: string
+  desc?: string
 }
 
-const PageHeader: React.FC<PageHeaderProps> = ({ title }) => {
+const PageHeader: React.FC<PageHeaderProps> = ({ title, desc, welcomeMessage, isHomePage = false }) => {
+  if (isHomePage) {
+    return (
+      <Box backgroundColor="gray08" as="header" padding={['3.1em 1em', null, null, '6.1em 3em']}>
```

Comment:
```suggestion
      <Box backgroundColor="gray08" as="header" px={['1em', null, null, '3em']} py={['3.1em', null, null, '6.1em']}>
```

Lebih readable kalau padding X sama padding Y dipisah.